pages/contribute/rule-feedback.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+If you "watch" the repository, you will get notifications when changes are happening.
+
+## Feedback for ideas and early drafts (Issues on GitHub)
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
+
+The top comment on an issue should hold the updated rule or algorithm, and underneath there will be a thread of comments and a history of actions related to the issue.
+
+Please feel free to add your own comments to issues.
+
+Start giving feedback on issues:
+
+- See the [ideas and early drafts that might need input](https://github.com/act-rules/act-rules.github.io/issues)
+
+- Use the [Definition of "Done"](../design/definition-of-done) to guide your review
+- Learn more about [commenting on issues](#) (TBD)
+
+## Feedback and reviews for draft rules (Pull Requests on GitHub)
+
+Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
```

Comment:
This makes it look like that before starting a review, one has to "register" for it, when all that is required is to have a GitHub account.

Reply:
Does someone need to be a member of the community group to review a PR, or is this still just GitHub permission levels?

Reply:
```suggestion
Pull requests are rule drafts that are ready for peer reviews. You can assign yourself as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to review it. However, even if three people are already reviewing it, we can always use more eyes on things.
```